here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -2027,6 +2044,9 @@ components:
                     maxItems: 3
                     minItems: 2
                   minItems: 4
+                  maxItems: 10000000
+                maxItems: 10000000
+              maxItems: 10000000
```

Comment:
A limit of 1 million means that we don't have any hard limit, I assume. But I would prefer to reduce nested limits like this a bit in case of unforeseeable overflow. A multipolygon consisting of 1 million parts, each having almost 1 million holes, might just be too huge to be in a HTTP request or response.

Reply:
Intention is not to keep it open to infinite value. It can be adjusted in future, when practical max limit is better known.